source/deployment/cluster.rst

```diff
@@ -134,75 +144,143 @@ A sample configuration for NGINX is provided below.  It assumes you have two Mat
 
------
+Update Guide
+------------
+
+Updating configuration changes while operating continuously
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A service interruption is not required for most configuration updates. See `Updating server version while operating continuously`_ for a list of configuration updates that require a service interruption.
+
+You can apply updates during a period of low load, but if your HA cluster is sized correctly, you can do it at any time. The system downtime is brief, and depends on the number of Mattermost servers in your cluster. Note that you are not restarting the machines, only the Mattermost server applications. A Mattermost server restart generally takes about 5 seconds.
+
+1. Make a backup of your existing ``config.json`` file.
+2. For one of the Mattermost servers, make the configuration changes to ``config.json`` and save the file. Do not reload the file yet.
+3. Copy the ``config.json`` file to the other servers.
+4. Shut down Mattermost on all but one server.
+5. Reload the configuration file on the server that is still running. Go to **System Console > Configuration** and click **Reload Configuration from Disk**
+6. Start the other servers. 
+
+Updating server version while operating continuously
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A service interruption is not required for security patch dot releases of the Mattermost server.
+
+You can apply updates during a period of low load, but if your HA cluster is sized correctly, you can do it at any time. The system downtime is brief, and depends on the number of Mattermost servers in your cluster. Note that you are not restarting the machines, only the Mattermost server applications. A Mattermost server restart generally takes about 5 seconds.
+
+1. Review the upgrade procedure in the *Upgrade Enterprise Edition* section of :doc:`../administration/upgrade`.
+2. Make a backup of your existing ``config.json`` file.
+3. Set your proxy to move all new requests to a single server. If you are using NGINX and it's configured with an upstream backend section in ``/etc/nginx/sites-available/mattermost`` then comment out all but the one server that you intend to update first, and reload NGINX.
+4. Shut down Mattermost on each server except the one that you are updating first.
+5. Upgrade each Mattermost instance that is shut down.
+6. On each server, replace the new ``config.json`` file with your backed-up copy.
+7. Start the Mattermost servers.
+8. Repeat the upgrade procedure for the server that was left running.
+
+Updating during scheduled maintenance downtime
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A service interruption is required when the update includes a change to the database schema or when a change to ``config.json`` requires a server restart, such as when making the following changes:
+
+  * Default Server Language
```

Comment:
Probably ok to not link. The people doing the upgrades have either changed config.json already and are familiar with the settings, or know that config.json was not changed.

I think links to other docs need to add significant value because they are not resilient in the sense that they can get unwittingly broken by changes to the target doc. A broken link is often worse than no link.